### ISSUE TITLE
Getting warning when installing via yarn

### ISSUE THREAD

**Issue by rahul3103** [rohanray/next-fonts#19]
yarn add next-fonts
```
yarn add v1.19.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "next-fonts > file-loader@2.0.0" has unmet peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
warning "next-fonts > url-loader@1.1.2" has unmet peer dependency "webpack@^3.0.0 || ^4.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 2 new dependencies.
info Direct dependencies
└─ next-fonts@0.19.0
info All dependencies
├─ next-fonts@0.19.0
└─ url-loader@1.1.2
✨  Done in 4.33s.
```

**Comment by edgarssilva** [comment]
Same problem

**Comment by rohanray** [comment]
Resolved in v 1.5.1
